docs/imaging/dlc.rst

```diff
@@ -0,0 +1,114 @@
+  
+* ``TrackingRaw.DLCPart``
+
+  Stores the raw state vector calculated via the method identifed in ``DLCTrackingProcessingMethod``
+
+* ``Tracking.DLCPart``
+
+  Stores the processed state vector, i.e. after conversion to real units, smoothing, and derivation of,e.g., speeds, angles etc. 
+
+* ``DLCPrediction``
+
+  Handles automated processing of a session
+
+
+
+Processing Methods
+-----------------------
+
+Initial work with DLC within the Moser group has focused on duplicating the same tracking methods used in the past - tracking the position and angle of the head of the subject. Typically, this took the form of tracking the left and right ears, instead of LEDs mounted to the subject's head. Consequently, exactly the same calculation method could be used, with a pre-calculation stage of identifying what data corresponds to each "LED".
+
+Future methods may be added, including potentially 3D information, but these methods must first be written.
+
+
+
+DLC Models
+--------------
+
+1. In DeepLabCut model directory hierarchy, accessing the model file requires:
+
+  * cfg (fields: Task | date | iteration)
+  * shuffle (not in config)
+  * trainingset_fraction - not in config, compute as: cfg['TrainingFraction'][trainingsetindex]
+  
+  (https://github.com/DeepLabCut/DeepLabCut/blob/326e6f7e48b70bee9360ba84a8ffc1772f50cbe3/deeplabcut/utils/auxiliaryfunctions.py#L451)
+
+2. Minimal setting for triggering DLC analysis is as follows:
+
+      ``deeplabcut.analyze_videos(cfg, video_paths, shuffle, trainingsetindex)``
+      
+  (https://github.com/DeepLabCut/DeepLabCut/blob/master/deeplabcut/pose_estimation_tensorflow/predict_videos.py)
+
+3. Generated Scorer/network name
+
+  * Take an example: "DLC_resnet50_mouse_openfieldJun30shuffle1_1030000"
+  * Format: ``DLC_`` + netname (``resnet50``) + cfg['Task'] (``mouse_openfield``) + cfg['date'] (``June30``) + shuffle + trainingsiterations
+  * Note that "trainingsiterations" here is not the same as "dlc_iteration" (which is more of a model version)
+  * "trainingsiterations" is inferred based on: cfg['snapshotindex']
+  * This scorer name is not sufficiently unique because it misses information about the dlc_iteration
+  
+  (https://github.com/DeepLabCut/DeepLabCut/blob/326e6f7e48b70bee9360ba84a8ffc1772f50cbe3/deeplabcut/utils/auxiliaryfunctions.py#L524)
+
+In summary, the combination of (dlc_task, dlc_date, dlc_iteration, dlc_shuffle, dlc_trainingsetindex, dlc_snapshotindex) is sufficient to uniquely specify one DLC model
+
+
+
+Workflow
+--------------
+
+DLC is not currently supported via the web gui, and requires handling via code. Sample notebooks for model ingestion are provided:
+  
+  1. `DLC model insertion <https://github.com/kavli-ntnu/dj-moser-imaging/blob/master/Helper_notebooks/DLC%20model%20insertion.ipynb>`_
+  2. `SessionDLC insertion <https://github.com/kavli-ntnu/dj-moser-imaging/blob/master/Helper_notebooks/Insert%20Session%20DLC.ipynb>`_
```

Comment:
No, that's handled via the webGUI! ( the second point). The first point (Model insertion) should be reserved to admins at the moment because of the naming / ... complexity.

Reply:
(i.e. the "Insert Session DLC.ipynb" is obsolete now that the web GUI supports it)